### ISSUE TITLE
Change the default size of the popup instance-translation window 

### ISSUE THREAD

**Issue by kasraavand** [goldendict/goldendict#1662]
Please consider adding an option for changing the default window size of the Scan pop up, I can't seem to be able to find that option anywhere in the preferences. So if such functionality already exists pls leme know.

**Comment by vedgy** [comment]
The functionality does exist. It is documented in GoldenDict's built-in help. Resizing is also described under #1010.

**Comment by kasraavand** [comment]
Yes, I figured out the temporary solution but it restarts after each system reboot. I was wondering if there's any permanent solution. I couldn't find that in config file either, btw.

**Comment by vedgy** [comment]
I think the `popupWindowGeometry` entry in GoldenDict config stores this info. Is the window geometry reset only after each system reboot or each time GoldenDict is restarted too? Perhaps this geometry isn't stored properly when your system is restarted, because your window manager doesn't support session management well. What if you resize the scan popup window, restart GoldenDict, then reboot the system?

**Comment by kasraavand** [comment]
the popupWindowGeometry hash changes after I update the window size manually but after rebooting regardless of restarting the goldendict or not it comes back to the default setting. It's as if somehow the config is regenerated with the default setting.

**Comment by vedgy** [comment]
What's your operating system and version? Desktop environment and/or window manager? GoldenDict version?

Are other GoldenDict settings preserved between reboots? Including the main window size and position?

**Comment by kasraavand** [comment]
Golden dict is 1.5.0-RC2+git, and OS Ubuntu 22.04.2 LTS with X11 as the windowing manager. All the settings such as font size are applied after reboot/reset just the window size doesn't. If you're sure that this isn't something explicitly happening in the code, i.e enforcing the default size and resetting the popupWindowGeometry tag, it might be happening implicitly. Either way, It's not that much of a bd IT might also be something related to the OS or some incompatibility between the version.

**Comment by vedgy** [comment]
X11 is not a window manager :). I guess it's GNOME desktop environment, which is the default in Ubuntu.

Is `mainWindowGeometry` reset to default after reboot? If not, then the issue is likely not caused by your desktop environment, but is in GoldenDict itself. In that case, perhaps the recent commits 74aa95220d04b8e4704ad6dac0c4c018f8e76f1a and e43c0d252130b0b83742d1847074df8f236d12cb will fix your scan popup geometry issue once you update to GoldenDict 1.5.
